eng/SnippetGenerator/Program.cs

```diff
@@ -16,9 +16,18 @@ public class Program
 
         public void OnExecuteAsync()
         {
-            foreach (var sdkDir in Directory.GetDirectories(BasePath))
```

Comment:
As we talked we should file an issue to move the SnippetGenerator to the azure-sdk-tools repo. In the world were folks set this new setting we will have race conditions with people trying to build this project and it is generally a bad idea to build tools that are also being consumed by the build as part of the build.

Reply:
Issue created here https://github.com/Azure/azure-sdk-for-net/issues/13679